images-list

```diff
@@ -3,6 +3,8 @@
 # https://github.com/rancher/image-mirror/pull/27
 #
 # LIST FORMAT: <SOURCE> <DESTINATION> <TAG>
+appscode/kubed rancher/kubed v0.13.2
```

Comment:
Should this image name be rancher/appscode-kubed? Seems that is the convention

Reply:
Indeed. I thought I had fixed them all compared to my first commit. Apparently not. Will fix this. And the others as well.

Thanks for checking.

Reply:
Cross-checking it seems that you looked at a single specific commit of the PR, namely https://github.com/rancher/image-mirror/pull/239/commits/42fab3e9f6be334e28b5f2e980943309b28d65ed

This is just the first commit of two in the PR.

When I look at the entire PR instead, i.e. https://github.com/rancher/image-mirror/pull/239/files I see
```
appscode/kubed rancher/appscode-kubed v0.13.2
appscode/kubed rancher/mirrored-appscode-kubed v0.13.2
```
i.e. as it should be. Same for all the other places you noted.

The second commit, https://github.com/rancher/image-mirror/pull/239/commits/a7074321c83b8c865a18bb2fc8117c1938f00288, fixed the mistakes I made in the first, and further added some images I had missed in my first round.

As such I consider your comments as resolved.